### PR TITLE
CO-3051 add activity to recurring_contracts

### PR DESCRIPTION
**Pull request by josueBulle**
related:
https://github.com/CompassionCH/compassion-modules/pull/1293
https://github.com/CompassionCH/compassion-switzerland/pull/1016

### CODE REVIEW

**Comment by ecino** [comment]
Can you add OCA repository for mis_builder module so that travis goes green please?

File: mis_builder_spn_event_info/__manifest__.py [hunk 1]
```diff
@@ -18,6 +18,6 @@
         'security/mis_spn_event_info.xml',
         'views/mis_spn_event_info.xml',
     ],
-    'installable': False,
+    'installable': True,
```
Comment (ecino) [inline mis_builder_spn_event_info/__manifest__.py]: This should not be present in this PR

**Review by ecino — CHANGES_REQUESTED** [review]

**Comment by ecino** [comment]
After reflextion, can you move modules mis_builder* into compassion-switzerland repository ? It makes more sense there I think.

**Review by ecino — CHANGES_REQUESTED** [review]
I think you removed the class inheritance by mistake.

**Comment by josueBulle** [comment]
The model `recurring_contract` was already inheriting `mail.activity.mixin`

**Review by ecino — APPROVED** [review]

**Pull request merged**